tyssue/geometry/bulk_geometry.py

```diff
@@ -38,6 +39,10 @@ def update_all(cls, eptm):
     def update_dcoords(eptm):
         SheetGeometry.update_dcoords(eptm)
 
+    @staticmethod
+    def update_ucoords(eptm):
+        SheetGeometry.update_ucoords(eptm)
+
```

Comment:
I don't know why it's there, inheritance should be enough (both for `update_dcoords` and `update_ucoords`), could you just delete those lines?